src/main/scala/SBTS3Resolver.scala

```diff
@@ -79,7 +82,22 @@ object SbtS3Resolver extends AutoPlugin {
 
       // convenience method, to use normal bucket addresses with `at`
       // without this resolver: "foo" at s3("maven.bucket.com").toHttps(s3region.value)
-      def toHttps(region: String): String = s"""https://s3-${region}.amazonaws.com/${url.stripPrefix("s3://")}"""
+      def toHttps(region: String): String = {
+        val bucketPath = url.stripPrefix("s3://")
+        val euCentral = Region.EU_Frankfurt.toString
+
+        //Correcting for fact that region enum resolves to null when provided US_STANDARD (us-east-1)
+        val correctedRegion = if(region == null) "us-east-1" else region
+
+        /*
+        Constructing host name based on region name. Frankfurt host name must be constructed using s3.{region}
+        for java sdk to sign its request using the v4 signature method (Frankfurt only supports v4 signing)
+         */
+        region match {
+          case euCentral => s"""https://s3.${correctedRegion}.amazonaws.com/${bucketPath}"""
+          case _ => s"""https://s3-${correctedRegion}.amazonaws.com/${bucketPath}"""
+        }
```

Comment:
Thanks for pointing it out, I didn't know that these patterns vary. Now looking through the docs I see that it's also different for China (having `.cn` suffix). So I want to find a way to get this prefix by the AWS SDK means.